src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1647,6 +1647,9 @@ void Unit::CalcAbsorbResist(Unit* victim, SpellSchoolMask schoolMask, DamageEffe
 
     RoundToInterval(auraAbsorbMod, 0.0f, 100.0f);
 
+    auto absorb_ignoring_damage = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
```

Comment:
@P-Kito : got a suggestion for how to replace the code in the line? I presume we will still be using
```cpp
[...] = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
```
but what to insert at the start of the line? Or is it OK just to remove `auto` like this:
```cpp
absorb_ignoring_damage = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
```